### PR TITLE
Preparing release 2.11.0

### PR DESCRIPTION
**Pull request by Natim**
**Tooling requirements**
* ``pip install zest.releaser[recommended]`` (Install zest.releaser, wheel and twine)

**Step 1**
```
$ git co -b prepare-X.Y.Z
$ prerelease
$ vim cliquet_docs/conf.py
$ rm -fr .venv/
$ make build-requirements
$ git ci -a --amend
$ git push origin prepare-X.Y.Z
```
* [x] Update Contributors file ``git shortlog -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' | sort``
* [x] Merge remaining pull requests
* [x] Update changelog
* [x] Update version in ``cliquet_docs/conf.py``
* [x] Update known good set of dependencies in requirements.txt

**Step 2**
```
$ release
$ git co master
$ git merge --no-ff prepare-X.Y.Z
$ postrelease
```
* [ ] Tag vX.Y.Z
* [ ] Publish on Pypi

**Step 3**
* [ ] Close milestone in Github
* [ ] Create next milestone in Github
* [ ] Add entry in Github release page
* [ ] Configure the version in ReadTheDocs
* [ ] Send mail to ML

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -37,22 +31,27 @@ This document describes changes between each past release.
   number of threads waiting for a connection (#509)
 - Add ``impacted_records`` attribute on ``ResourceChanged`` event (#501)
   This also allows listeners to react on particular field change, since old and
-  new version of records is provided.
+  new version of records is provided.)
 
-**Bug fixes**
+**Bugs fixes**
```
Comment (leplatrem) [inline CHANGELOG.rst]: I think we say *bug fixes*

File: CHANGELOG.rst [hunk 2]
```diff
@@ -37,22 +31,27 @@ This document describes changes between each past release.
   number of threads waiting for a connection (#509)
 - Add ``impacted_records`` attribute on ``ResourceChanged`` event (#501)
   This also allows listeners to react on particular field change, since old and
-  new version of records is provided.
+  new version of records is provided.)
```
Comment (leplatrem) [inline CHANGELOG.rst]: where is this parenthesis open?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -24,12 +24,6 @@ This document describes changes between each past release.
 - Add the ``protocol_version`` to tell which protocol version is
```
Comment (leplatrem) [inline CHANGELOG.rst]: maybe specify in hello URL view?

File: CHANGELOG.rst [hunk 4]
```diff
@@ -24,12 +24,6 @@ This document describes changes between each past release.
...
   implemented by the service. (#324)
 
-**Breaking changes**
-
-- For PostgreSQL backends, it is recommended to specify ``postgresql://``
-
```
Comment (leplatrem) [inline CHANGELOG.rst]: why move breaking changes below ?

File: CHANGELOG.rst [hunk 5]
```diff
@@ -37,22 +31,27 @@ This document describes changes between each past release.
...
   This also allows listeners to react on particular field change, since old and
-  new version of records is provided.
+  new version of records is provided.)
 
 **Bug fixes**
 
-- Fix Service CORS not being set when plugins are included
-- Fix crash with Redis backend if record parent/id is unicode (fixes #556)
-- Fix principals of permission backend not being plugged by default (#573)
-- Fix Redis error traces not being logged (#560)
```
Comment (leplatrem) [inline CHANGELOG.rst]: Why remove those?

File: CHANGELOG.rst (hunk 4 above)
Comment (Natim) [inline CHANGELOG.rst]: I just merged the two New Features paragraph, but I can put Breaking change before new features if you'd like to.

File: CHANGELOG.rst (hunk 5 above)
Comment (Natim) [inline CHANGELOG.rst]: I probably did a mistake in my IDE at some point, strange.

File: CHANGELOG.rst (hunk 3 above)
Comment (Natim) [inline CHANGELOG.rst]: Ok done below.

**Comment by leplatrem** [comment]
r+!

**Pull request merged**